packages/flutter/test/material/data_table_test.dart

```diff
@@ -94,9 +106,37 @@ void main() {
     ));
     await tester.pumpAndSettle(const Duration(milliseconds: 200));
 
-    await tester.tap(find.text('375'));
+    await tester.tap(find.text('375'), pointer: 2);
```

Comment:
Could you explain the use of the pointer parameter here? I'm still not clear what its purpose is :)